cardinal/router/iterator/iterator.go

```diff
@@ -48,7 +48,8 @@ func New(
 	}
 }
 
-// Each iterates over txs
+// Each iterates over txs from the data availability layer. For each batch of transactions found in
+// each tick, it will apply the callback function to that batch and it's respective tick and timestamp.
```

Comment:
```suggestion
// Each iterates over txs from the base shard. For each batch of transactions found in
// each tick, it will apply the callback function to that batch and it's respective tick and timestamp.
```

technically doesn't do DA reading